README.md

```diff
@@ -30,6 +30,7 @@ Options:
   -p, --port                Port to listen on                           [number]
   -r, --rate-limit-enabled  Enable rate limiting                       [boolean]
   -c, --cors                CORS whitelist origins                [string]
+  -S, --studio              Start Soul Studio in parallel              [boolean]
```

Comment:
this is the only upper-case shortcut option, though the full option is still lower-case .  Do we need that slight complexity?